app/(candidate)/onboarding/[slug]/[step]/components/ballotOffices/BallotRaces.js

```diff
@@ -197,54 +196,16 @@ export default function BallotRaces(props) {
         count={numOfRaces || 0}
       />
       <Sticky top={56} innerZ={10}>
-        <div className="bg-white pt-4 pb-2">
-          <div className="grid grid-cols-12 gap-4">
-            <div className="col-span-12 lg:col-span-6 pt-1">
-              <TextField
-                label="Search"
-                value={inputValue}
-                fullWidth
-                onChange={(e) => setInputValue(e.target.value)}
-              />
-            </div>
-            <div className="col-span-12 lg:col-span-4">
-              <Select
-                className="py-[2px]"
-                native
-                required
-                variant="outlined"
-                fullWidth
-                value={level}
-                onChange={(e) => setLevel(e.target.value)}
-              >
-                <option value="">Election Level</option>
-                {['local', 'county', 'state', 'federal'].map((value) => (
-                  <option value={value} key={value}>
-                    {value} office
-                  </option>
-                ))}
-              </Select>
-            </div>
-            <div className="col-span-12 lg:col-span-2">
-              <Select
-                className="py-[2px]"
-                native
-                required
-                variant="outlined"
-                fullWidth
-                value={yearFilter}
-                onChange={(e) => setYearFilter(e.target.value)}
-              >
-                <option value="">Year</option>
-                {electionYears.map((value) => (
-                  <option value={value} key={value}>
-                    {value}
-                  </option>
-                ))}
-              </Select>
-            </div>
-          </div>
-        </div>
+        <OfficeSelectionFilters
+          {...{
+            electionYears,
+            onChange: ({ inputValue, level, yearFilter }) => {
+              setInputValue(inputValue);
+              setLevel(level);
+              setYearFilter(yearFilter);
+            },
+          }}
```

Comment:
please pass props the same way we do in the rest of the project
electionYears={electionYears}
onChange={}